src/main/java/net/dv8tion/jda/core/entities/Game.java

```diff
@@ -100,15 +159,15 @@ static Game of(String name, String url)
      *
      * @return True if the provided url is valid for triggering Discord's streaming status
      */
-    static boolean isValidStreamingUrl(String url)
+    public static boolean isValidStreamingUrl(String url)
     {
-        return url != null && url.matches("^https?:\\/\\/(www\\.)?twitch\\.tv\\/.+");
+        return url != null && url.matches("^https?://(www\\.)?twitch\\.tv/.+");
```

Comment:
I see that this regex changed. Was it supposed to?

Reply:
Yes this was intended.